magpie/api/login/login.py

```diff
@@ -210,8 +210,7 @@ def authomatic_login(request):
             cred = Credentials(authomatic_handler.config, token=access_token, token_type=token_type, provider=provider)
             provider.credentials = cred
             result = LoginResult(provider)
-            # noinspection PyProtectedMember
-            result.provider.user = result.provider._update_or_create_user(data, credentials=cred)
+            result.provider.user = result.provider._update_or_create_user(data, credentials=cred)  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _update_or_create_user of a client class](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)